frontend/src/Components/Authentication/Authentication.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import { Component } from 'react';
+import authService from '../../services/authService'
+import Input from '@material-ui/core/Input'
+import InputLabel from '@material-ui/core/InputLabel'
+import Button from '@material-ui/core/Button'
+import Grid from '@material-ui/core/Grid'
+import Paper from '@material-ui/core/Paper'
+import FormControl from '@material-ui/core/FormControl'
+import Fade from '@material-ui/core/Fade';
+import { withSnackbar } from 'notistack';
+import './Authentication.css'
```

Comment:
You could use css modules to be a bit more consistent with style handling across the app.